content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,226 @@
+
+1. ###### Log on to the backend server as wodadmin and update the notebook repository.
+2. ###### R﻿un a wod-deliver to update the relevant backend server.
+
+   ```
+   git remote update
+   git rebase
+   wod-deliver
+   ```
+
+T﻿he very same processes will apply to the move to production phase.
+
+# Complex workshop example:
+
+II will not repeat the steps I descirbed earlier for the simple workshop example. I will focus here on the specific aspects related to this  new workshop. Are you familiar with High Performance Computing (HPC)? I am not. Even tough I am surrounded by some experts in that field in the HPE Grenoble Office. Let's consider that one of these colleagues is willing to build up a dedicated workshop on HPC Stax. As usual, we will start by a meeting each of us will explain to the other his goals, and how to achieve them. Once I get a clearer understanding of the technology involved, he and I can move on figure out the best platform to run his woorkshop on.
+
+A﻿s an admin of the Workshops-on-demand infrastructure, I had to perform several tasks:
+
+##### O﻿n the backend server:
+
+t﻿he  workshop will require:
+
+* A dedicated server running docker to host the student containers in which the workshops' labs  will take place.
+
+  * This means preparing a server (VM or physical) : We will consider it as an appliance.
+  * Updating the relevant variable file to associate the IP address of the server to the workshop (there could be multiple servers too associated to a given workshop)
+* A﻿ set of scripts under wod-backend/scripts or wod-private/scripts folders depending on the nature of the workshop to manage the workshop's lifecycle.
+
+  * Some generic scripts applying to all workshops' appliances : general setup phase setting up common requirements for any appliance (student users creation, ssh keys, etc..) up to some specific ones dedicated to a given workshop.
+
+    * create-appliance.sh (ssh keys, ldap setup)
+    * setup-appliance.sh \[WKSHP-NAME] (Student setup, Appliance Setup, Workshop setup) calls:
+
+      * setup-globalappliance.sh (global / generic setup)
+      * setup-\[WKSHP-NAME].sh (Prepare appliance with workshop's reqs, Docker image for instance)
+    * create-\[WKSHP-NAME].sh (called at deployement time to instantiate the necessary appliance(s) requiered by the workshop 
+    * reset-appliance (reset ssh keys and students credentials on appliance
+    * cleanup-\[WKSHP-NAME].sh (takes care of cleanup some workshop's specifics)
+    * reset-\[WKSHP-NAME].sh (reset of the workshop's appliance, docker compose down of a container for instance)
+* A set of variables to be leveraged by the notebooks. These variables are to be set in yml format. They will be parsed at deployment times to set student ids, appliance IP addresses, and other relevant parameters like ports, simulated hardware information for instance.
```

Comment:
The last part of the last sentence has me confused. I'm not sure what you're trying to say.